<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>988</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Control de Alumnos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Matrícula:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMatricula">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>108</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Datos Personales</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPaterno">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Paterno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Materno</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaterno">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Sexo</string>
    </property>
    <widget class="QRadioButton" name="rdbMasculino">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>106</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Masculino</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbFemenino">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>106</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Femenino</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dirección</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Calle y número:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDireccion">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Municipio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMunicipio">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>560</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbEstado">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
    <item>
     <property name="text">
      <string>Aguascalientes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Baja California</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Baja California Sur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Campeche</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chiapas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chihuahua</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ciudad de México</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coahuila</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Durango</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guanajuato</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guerrero</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hidalgo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jalisco</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Edomex</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Michoacan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Morelos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nayarit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nuevo León</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oaxaca</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Puebla</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Queretaro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quintana Roo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>San Luis Potosí</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sinaloa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sonora</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tabasco</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tamaulipas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tlaxcala</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Veracruz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yucatán</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zacatecas</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Información Académica</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>690</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrera:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbCarrera">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>680</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>Licenciado en Gestión de la Responsabilidad Social</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Licenciado en Contador Público</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Licenciado en Administración</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Licenciado en Negocios Internacionales</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Licenciado en Tecnologías de Información</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>740</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Materias Preferidas</string>
    </property>
    <widget class="QCheckBox" name="ckbContabilidad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Contabilidad</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckbEstadistica">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Estadística</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckbProgramacion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Programación</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckbBD">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Base de Datos</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckbOperaciones">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Operaciones</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>740</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Beca</string>
    </property>
    <widget class="QRadioButton" name="rdb100">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdb80">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>80%</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdb50">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>50%</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbNA">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Sin beca</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>900</y>
      <width>108</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>900</y>
      <width>108</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Información</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPantalla">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>170</y>
      <width>651</width>
      <height>311</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>490</y>
      <width>108</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>557</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registros</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbwTabla">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>590</y>
      <width>641</width>
      <height>291</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Matrícula</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Paterno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Materno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sexo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Edad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dirección</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Municipio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Carrera</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Beca</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Materias</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>890</y>
      <width>108</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReporte">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>890</y>
      <width>141</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Reporte</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>380</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Edad:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEdad">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>373</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>64</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Acción:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbAccion">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>83</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>Registrar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnAplicar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>108</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cmbAccion</tabstop>
  <tabstop>btnAplicar</tabstop>
  <tabstop>txtMatricula</tabstop>
  <tabstop>btnBuscar</tabstop>
  <tabstop>txtNombre</tabstop>
  <tabstop>txtPaterno</tabstop>
  <tabstop>txtMaterno</tabstop>
  <tabstop>rdbMasculino</tabstop>
  <tabstop>rdbFemenino</tabstop>
  <tabstop>txtEdad</tabstop>
  <tabstop>txtDireccion</tabstop>
  <tabstop>txtMunicipio</tabstop>
  <tabstop>cmbEstado</tabstop>
  <tabstop>cmbCarrera</tabstop>
  <tabstop>rdb50</tabstop>
  <tabstop>rdb100</tabstop>
  <tabstop>rdb80</tabstop>
  <tabstop>rdbNA</tabstop>
  <tabstop>ckbContabilidad</tabstop>
  <tabstop>ckbEstadistica</tabstop>
  <tabstop>ckbProgramacion</tabstop>
  <tabstop>ckbBD</tabstop>
  <tabstop>ckbOperaciones</tabstop>
  <tabstop>btnRegistrar</tabstop>
  <tabstop>btnActualizar</tabstop>
  <tabstop>btnLimpiar</tabstop>
  <tabstop>btnEliminar</tabstop>
  <tabstop>btnReporte</tabstop>
  <tabstop>txtPantalla</tabstop>
  <tabstop>tbwTabla</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
